cfme/cloud/provider/openstack.py

```diff
@@ -82,12 +84,17 @@ def deployment_helper(self, deploy_args):
 
     @classmethod
     def from_config(cls, prov_config, prov_key):
-        endpoints = {}
-        endpoints[RHOSEndpoint.name] = RHOSEndpoint(**prov_config['endpoints'][RHOSEndpoint.name])
+        endpoints = {
+            RHOSEndpoint.name: RHOSEndpoint(**prov_config['endpoints'][RHOSEndpoint.name])
+        }
 
-        endp_name = EventsEndpoint.name
-        if prov_config['endpoints'].get(endp_name):
-            endpoints[endp_name] = EventsEndpoint(**prov_config['endpoints'][endp_name])
+        event_endpoint_config = prov_config['endpoints'].get(EventsEndpoint.name, {})
```

Comment:
@digitronik changed this up a bit to explicitly check for AMQP (yes I realize that's really the only event endpoint type :) )

Let me know what you think, I've tested it with AMQP and a bogus event stream type in yamls, it logged correctly and included the bogus endpoint in the provider object (won't configure, but just testing if a new type is added).

Reply:
@mshriver I like `logger` with `warning` :1st_place_medal: